### PR TITLE
LPS-200407 Refactor all implementations of Autocomplete component to use SingleSelect

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://liferay.atlassian.net/browse/LPS-200407

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:object

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by rebsilva** [comment]
i just started reviewing the FE part! :cherry_blossom:

**Review by rebsilva — APPROVED** [review]
LGTM! :tada: Great work @danvb112 :cherry_blossom:

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsDetailsDisplayContext.java [hunk 1]
```diff
@@ -173,20 +172,29 @@ public List<KeyValuePair> getScopeKeyValuePairs(String scope) {
 				_panelCategoryRegistry.getChildPanelCategories(
 					panelCategoryKey);
 
+			JSONArray itensJSONArray = JSONFactoryUtil.createJSONArray();
```
Comment (marcelabc) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsDetailsDisplayContext.java:175]: We should rename this variable to ite**m**sJSONArray :+1:

**Comment by andreefarias** [comment]
I'll close this PR, since that I sent a new PR with the refactor from testcases [here](https://github.com/liferay-objects/liferay-portal/pull/2843).

cc: @danvb112 @rebsilva

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Pull request closed without merging**